src/Behat/Behat/ApplicationFactory.php

```diff
@@ -47,6 +48,10 @@
  */
 final class ApplicationFactory extends BaseFactory
 {
+    /**
+     * @deprecated this constant will not be updated for releases after 3.13.0 and will be removed in the next major.
+     * You can use composer's runtime API if you need to check the installed behat version.
```

Comment:
I would probably change this second line to `Call the getVersion() function in this class to get the installed Behat version using composer's runtime API`

Reply:
Ah, yes of course, missed the point that getVersion() has protected visibility. Your new suggestion sounds great

Reply:
@carlos-granados done